### PR TITLE
feat: node power

### PR DESCRIPTION
**Pull request by renauter**
Related to:

- #88

### CODE REVIEW

**Comment by renauter** [comment]
Tested on local chain
Node 1 and Node 2 belong to Farm 1

photo_2023-02-01_17-51-48

File: src/types/v125.ts [hunk 1]
```diff
@@ -0,0 +1,359 @@
+import type { Result } from './support'
```
Comment (DylanVerstraete) [inline src/types/v125.ts:1]: Can you rework this file to only contain the changes required for power ?

**Review by DylanVerstraete — APPROVED** [review]
LGTM, Already approving

**Comment by DylanVerstraete** [comment]
Please also add following: rework the database migration to set the default value of power target and state to "up"

File: src/types/v125.ts (hunk 1 above)
Comment (renauter) [reply src/types/v125.ts:1]: Ok

Btw, all the `src/types/v*.ts` filesstart with this line: 
~~~rust
import type { Result } from './support'
~~~
which is never used 
`'Result' is declared but its value is never read.`

Should I remove this line in all files?

Comment (DylanVerstraete) [reply src/types/v125.ts:1]: Yes I think you can safely remove this

**Review by DylanVerstraete — APPROVED** [review]

**Pull request merged**